packages/calypso-build/README.md

```diff
@@ -0,0 +1,3 @@
+# Calypso build
```

Comment:
We should be consistent, either the literal package names `@automattic/calypso-build` or a prose name `Calypso Build`. These changes include one of each.

Reply:
Good catch. I switched all the readmes to use prose name.